packages/connect-examples/webextension-v3/update.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Utility script. Not part of example itself. It only helps put the example together
+ */
+
+const fs = require('fs');
+const path = require('path');
+const fetch = require('node-fetch');
+
+const vendorPath = path.join(__dirname, 'vendor');
+const inlineScriptPath = path.join(vendorPath, 'trezor-connect.js');
+const usbPermissionsScriptPath = path.join(vendorPath, 'trezor-usb-permissions.js');
+const contentScriptPath = path.join(vendorPath, 'trezor-content-script.js');
+
+if (!fs.existsSync(vendorPath)) {
+    fs.mkdirSync(vendorPath);
+}
+
+[inlineScriptPath, contentScriptPath, usbPermissionsScriptPath].forEach(path => {
+    if (fs.existsSync(path)) {
+        fs.rmSync(path);
+    }
+});
+
+fetch('https://connect.trezor.io/9/trezor-connect.js')
+    .then(response => response.text())
+    .then(text => fs.writeFileSync(path.join(__dirname, 'vendor', 'trezor-connect.js'), text));
+
+const srcPath = path.join(__dirname, '../../connect-web/src/webextension');
+
+fs.copyFileSync(path.join(srcPath, 'trezor-content-script.js'), contentScriptPath);
+fs.copyFileSync(path.join(srcPath, 'trezor-usb-permissions.js'), usbPermissionsScriptPath);
```

Comment:
I thinking, maybe `packages/connect-examples/webextension-v3/trezor-usb-permissions.html` should be part of this utility script too? 

also, would it make sense to move this utility script one level up? and reuse it between examples?

Reply:
I think that makes sense, what about changing the file structure a bit like:

```
packages/connect-examples/webextension/mv2/
packages/connect-examples/webextension/mv3/
packages/connect-examples/webextension/update.js
```

What do you think?

Reply:
Yes, that makes sense, I added it to the update script bf8fe9d214b3b829e15c94252afe9535c66258b2